Source/WebKit/UIProcess/API/APIHitTestResult.h

```diff
@@ -90,6 +90,10 @@ class HitTestResult : public API::ObjectImpl<API::Object::Type::HitTestResult> {
 
     bool allowsFollowingImageURL() const { return m_data.allowsFollowingImageURL; }
 
+    const std::optional<WebCore::ResourceResponse>& linkLocalResourceResponse() const {
```

Comment:
Either everything should be on the same line (which would be better in this case) or the { needs to be on the next line.